plugins/woocommerce-admin/client/two-column-tasks/completed-header-with-ces.tsx

```diff
@@ -0,0 +1,215 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { useEffect, useState } from '@wordpress/element';
+import { EllipsisMenu } from '@woocommerce/components';
+import { recordEvent } from '@woocommerce/tracks';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { OPTIONS_STORE_NAME, WCDataSelector, WEEK } from '@woocommerce/data';
+import { Button, Card, CardHeader } from '@wordpress/components';
+import { Text } from '@woocommerce/experimental';
+import {
+	CustomerFeedbackModal,
+	CustomerFeedbackSimple,
+} from '@woocommerce/customer-effort-score';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './completed-header-with-ces.scss';
+import HeaderImage from './completed-celebration-header.svg';
+
+type TaskListCompletedHeaderProps = {
+	hideTasks: () => void;
+	keepTasks: () => void;
+	showCES: boolean;
+};
+
+const ADMIN_INSTALL_TIMESTAMP_OPTION_NAME =
+	'woocommerce_admin_install_timestamp';
+const SHOWN_FOR_ACTIONS_OPTION_NAME = 'woocommerce_ces_shown_for_actions';
+const CES_ACTION = 'store_setup';
+
+function getStoreAgeInWeeks( adminInstallTimestamp: number ) {
+	if ( adminInstallTimestamp === 0 ) {
+		return null;
+	}
+
+	// Date.now() is ms since Unix epoch, adminInstallTimestamp is in
+	// seconds since Unix epoch.
+	const storeAgeInMs = Date.now() - adminInstallTimestamp * 1000;
+	const storeAgeInWeeks = Math.round( storeAgeInMs / WEEK );
+
+	return storeAgeInWeeks;
+}
+
+export const TaskListCompletedHeaderWithCES: React.FC< TaskListCompletedHeaderProps > = ( {
```

Comment:
Could this just be `TaskListCompletedHeader`?  The `WithCES` portion seems dependent on the `showCES` prop.